Gruntfile.js

```diff
@@ -123,7 +123,7 @@ module.exports = function (grunt) {
     grunt.registerTask('compile:js', function (fullCompile) {
         grunt.task.run(['clean:js', 'compile:inlineSvgs']);
 
-        grunt.task.run(['concurrent:requireJS', 'copy:javascript', 'uglify:javascript']);
+        grunt.task.run(['concurrent:requireJS', 'copy:javascript', 'concat:curlPlusCorePlusBoot', 'uglify:javascript']);
```

Comment:
or just `concat:standard`? what we're doing here, at this level, is creating the standard js. that it contains curl and boots enhanced etc is contingent, but that there is a `standard.js` isn't